lib/filestream/filestream.go

```diff
@@ -314,7 +314,5 @@ func putBufioWriter(bw *bufio.Writer) {
 var bwPool sync.Pool
 
 type streamTracker struct {
-	fd     uintptr
```

Comment:
Could you exclude this changes from PR? 

 It breaks linter, but it's better to create separate PR for it.

Reply:
accidentally commited this file